src/math/stats.cr

```diff
@@ -0,0 +1,11 @@
+module Math
+  extend self
+  
+  def mean(value : StaticArray | Array | Tuple)
+    value.sum / value.length
+  end
+
+  def mean(value : Enumerable)
```

Comment:
Well, if that's the case the pr should closed indeed. @asterite Do you confirm this?

I was under the impression we were keeping as much as possible in the core, for now.